src/odemis/driver/xt_client.py

```diff
@@ -1524,15 +1539,25 @@ def __init__(self, name, role, parent, channel="ion2", **kwargs):
 
     def prepareForScan(self):
         """
-        Make sure the scan mode is in "full_frame" and not in "external"
+        Make sure the scan mode is in "full_frame" and not in "external" before starting to scan.
         """
+        # Currently the FIB blanker is not supported via the XT interface,
+        # so we cannot blank the beam at the start and end of the scan.
+
+        # Note: ideally this would be based on an external VA, which when put to
+        # 'auto' would switch to full frame at the start of a scan.
```

Comment:
I think the FIB was actually autblanked via XT, otherwise we couldn't scan with it. It would be unblanked when swithcing to FLM.

Reply:
I think your comment should be for line 1544 and 1545? I based that comment on what was there previously:
```
# TODO: When the acquisition ends, if blanker is set to automatic (None), we need to activate
#  the blanker again. That should also happen when switching e-beam <--> FIB, but currently
#  the FIB blanker is not supported via the XT interface.
```
Reading that comment I concluded that FIB blanking is not supported at the moment.

I also think XT does some autoblanking for the FIB. If you are sure you know when it blanks and unblanks I can add that to the comment. Otherwise I can update the comment to:

```
# Currently the XT interface autoblanks the FIB, control of the FIB blanker is not supported
# so we do not blank the beam at the start and end of the scan.
```